Packs/IntegrationsAndIncidentsHealthCheck/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Integrations & Incidents Health Check",
     "description": "Do you know which of your integrations or open incidents failed? With this content, you can view your failed integrations and open incidents",
     "support": "xsoar",
-    "currentVersion": "1.1.10",
+    "currentVersion": "1.1.11",
```

Comment:
no need to bump version

Reply:
the build failed when I didnt bumped the version